app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/AccountCheckController.scala

```diff
@@ -36,7 +36,9 @@ class AccountCheckController @Inject()(
 
   def accountCheck(redirectUrl: String): Action[AnyContent] = authAction.async {
     implicit request =>
-      logger.info(s"[AccountCheckController][accountCheck] User IP: ${request.headers.get("True-Client-IP")}")
+      logger.info(
+        s"[AccountCheckController][accountCheck] User IP: ${request.headers.get("True-Client-IP")}"
```

Comment:
Can you delete this? I added this for some allow list testing when I was stuck, we likely don't need it anymore.

Reply:
deleted